input_buffer.cc

```diff
@@ -39,11 +39,11 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <limits>
 #include <functional>
 #ifndef NDEBUG
 #include <iostream>
 #endif
+#include <limits>
```

Comment:
commit message doesn't seem to match?

Reply:
Yes and no, FreeBSD added the change in https://github.com/freebsd/freebsd-src/commit/1c44d2bf86834a7214e0cc7bde7c956b5b02fd21.
It is only partially applied here, and I left the commit messages verbatim.

Reply:
Let me know if I should add an `Obtained from:` entry in each commit message.